src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends AbstractFigure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculatedArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
```

Comment:
There is a typo in the method name 'calculatedArea'. It should be 'calculateArea' to follow standard naming conventions and to match the expected method name in the Figure interface.